game-core/src/main/resources/META-INF/triplea/product.properties

```diff
@@ -1 +1 @@
-version = 1.10.@buildId@
+version = 2.0.0
```

Comment:
I'm not quite sure if I like this change.
I feel like at some point someone is going to increment the "patch" number in semver terms and wonder why everything breaks

Reply:
That would indicate a misunderstanding of what the numbers represent and how they are used.

We could leave off the last '.0' and append it with the build number. I like though having in dev the build number to look like it would in production and have the same number of digits. That is purely for consistency between dev and prod, to reduce differences  which means our testing of dev carries over more to production. Having behavior that is production only is risky and can't be tested by definition.